app/models/company_setting.rb

```diff
@@ -1,3 +1,11 @@
 class CompanySetting < ActiveRecord::Base
   belongs_to :company
-end
+
+  def self.create_company_settings?(company_settings_object)
```

Comment:
Naming convention, adding object at last is not needed it can be company_setting if single or company_settings if collection.